packages/react-components/react-select/src/components/Select/useSelect.tsx

```diff
@@ -13,15 +13,15 @@ import type { SelectProps, SelectState } from './Select.types';
  * @param ref - reference to the `<select>` element in Select
  */
 export const useSelect_unstable = (props: SelectProps, ref: React.Ref<HTMLSelectElement>): SelectState => {
-  const { select, icon, root, size = 'medium', appearance = 'outline', inline = false } = props;
+  const { select, icon, root, appearance = 'outline', inline = false, onChange, size = 'medium' } = props;
```

Comment:
Can you make a follow up PR to remove `inline`? We removed this from all the other input controls and I don't see it being used in Select's styles.

Reply:
I'll do that now 👍